provider/interface.go

```diff
@@ -6,12 +6,38 @@ import (
 	"github.com/pivotal-cf/brokerapi/domain"
 )
 
-//go:generate counterfeiter -o fakes/fake_service_provider.go . ServiceProvider
-type ServiceProvider interface {
-	Provision(context.Context, ProvisionData) (dashboardURL, operationData string, isAsync bool, err error)
```

Comment:
really happy to see these bag-of-strings return types go 🚮  👍

Reply:
me too ... 

```
return "", "", "", true, blah // yuk
```